Standards/CCSMB-4.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-4: Lua
+
+*Author: Oliver Cooper (oeed)*
+*Last updated: 2022-01-27*
+*Version: v1.0.0*
```

Comment:
these should be spaced out a little more

Reply:
```suggestion
*Author: Oliver Cooper (oeed)*

*Last updated: 2022-01-27*

*Version: v1.0.0*
```